README.md

```diff
@@ -16,3 +16,93 @@ https://github.com/dwyl/intellectual-property and raise an
 on GitHub if you require clarification!
 
 See: https://github.com/dwyl/contributing/issues/1 + https://github.com/dwyl/summer-2015/issues/27
+
+## Recommended Guidelines
+
+### Labels
+
+GitHub can be a _great_ project management tool if used _effectively_. One of the ways you can manage your project backlog is through **labels**. Labels are a great way to **organise** and **track** the progress of a given project at _any_ point in time.
+
+At dwyl we have created our _own_ standard list of labels that we use for each of our projects. This gives our developers the ability to move between projects **quickly** and **productively**, saving time and energy in the process.
+
+Here is our **[complete list](https://github.com/dwyl/contributing/labels)** of labels and how they should be used:
+
+![labels](https://cloud.githubusercontent.com/assets/12450298/18248682/afcd6974-7371-11e6-84bf-0cb9f4677d92.png)
+
+- `bug` #EE0701 - report a bug that needs fixed
+- `duplicate` #CCCCCC - duplicate issue
+- `enhancement` #84B6EB - improving existing code
+- `help-wanted` #128A0C - looking for help or expertise on a subject
+- `in-progress` #009688 - to be added when an issue has been assigned
+- `in-review` #128A0C - once a PR has been submitted relating to the issue
+- `invalid` #E6E6E6 - GitHub standard tag (not used)
+- `please-test` #08E700 - added after PR is merged (assign to product owner)
+- `priority-1` #0D47A1 - drop everything and work on this (used only when _completely_ neccessary)
+- `priority-2` #1976D2 - high priority issue (what needs doing now)
+- `priority-3` #42A5F5 - high priority (what needs doing next)
+- `priority-4` #8DC9F9 - low priority (to be upgraded later)
+- `priority-5` #C5DEF5 - lowest priority (non-urgent changes)
+- `question` #CC317C - for open questions and discussions
+- `T[x]d` #F06292 - time in 'x' days it should take (estimation)
+- `T[x]h` #F7C6C7 - time in 'x' hours it should take (estimation)
+- `technical` #D4C5F9 - technical issue for developers
+- `wontfix` #ffffff - when an issue won't be addressed (with reason why in comment)
```

Comment:
We need a way/process to add (_discuss_) new labels. 
Time has a few good ones: https://github.com/dwyl/time/labels but we need to _standardise_ them...